src/main/thrift/parquet.thrift

```diff
@@ -211,6 +204,93 @@ struct Statistics {
    4: optional i64 distinct_count;
 }
 
+/** Empty structs to use as logical type annotations */
+struct StringType {}
```

Comment:
I can add a comment about UTF8, but I don't think we should allow other encodings, unless there is a new encoding that becomes the best standard, like UTF8 is today.